cristin-commons/src/main/java/no/unit/nva/cristin/model/CristinQuery.java

```diff
@@ -46,6 +47,20 @@ public static String getUnitIdFromOrganization(String organizationId) {
         return extractLastPathElement(unitId);
     }
 
+    /**
+     * Creates a URI to Cristin project with specific ID and language.
+     *
+     * @param id       Project ID to lookup in Cristin
+     * @return an URI to Cristin Projects with ID
+     */
+    public static URI fromIdentifier(String id) {
```

Comment:
This should describe that it is actually a project id generator, since it it moved to a common class